### ISSUE TITLE
[W-Change]

### ISSUE THREAD

**Issue by VBPXKSMI** [Kornilov1/AI-Dungeon-2-Anons-modded-Versions-Colab#178]
This is related to a new "[Custom Prompts](https://github.com/VBPXKSMI/Open-CYOAI-Project/wiki/Custom-Promts)" page layout Anons asked for, basically is the same as the "[Custom Promts Index](https://github.com/VBPXKSMI/Open-CYOAI-Project/wiki/Custom-Prompts-Index)" and related pages we have now in use, the diference is that this new one combines everything under a single page, and I managed to find a way to make the editting process of it not that bothersome so it could work without much hassle.

What I wanted to know are your thoughts of it @MunasheDov, are you willing to maintaing that page updated? Or do you prefer the old layout instead? Only one of them could be used at the same time, to avoid missunderstandings, reason why it is currently "hidden" from view.

***

Another issue:
@frowo Recently made a fork of thadunge2 and is working on it, and he also intends to make a more simple and straightfoward Wiki of ours one, including a re-formatting of the "[Custom Prompts](https://github.com/frowo/LucidTeller/wiki/Custom-Prompts)" page (using the new layout as a basis).

The things is, he is also adding prompts to it from the threads, so this could cause some disorganisation regarding info along the way in the long run.
He is willing to work on the page on our Wiki to avoid conflicts, but since this falls directly under the task assigned to you I wanted to know your opinion on the whole matter first before giving him an answer.

***

Also I would like to hear of your whereabouts, it has been some time since you showed up.

Hope to hear from you soon.

**Comment by MunasheDov** [comment]
I noticed the single page version a while ago; it's fine with me since it makes searching for stuff faster. What was the easier editing method you found? I'm fine with Frowo editing the Wiki. I haven't really been around much since the threads seemed to die down a lot, and the editing format got a little more onerous than I'd like to regularly deal with. The whole need to separate out the prompt and context, with all the sugar to format things for every new prompt was a little annoying, and with the slowdown, by the time a new prompt appeared I'd have almost forgotten the formatting.

**Comment by MunasheDov** [comment]
The long processing time upon saving an edit is a bit of a downside for the single page style.

**Comment by VBPXKSMI** [comment]
@MunasheDov 
>What was the easier editing method you found?

I just added the tip for easier searching inside the file, that way you can navigate between section quite quickly, is not much but it helps.
>The long processing time upon saving an edit is a bit of a downside for the single page style.

Damn, you found that it was behaving this way? I thought I might have overreacted when I said this the first time, and when I created the page didn't see that much of a difference, but if you say it still has it...

>the editing format got a little more onerous than I'd like to regularly deal with. The whole need to separate out the prompt and context, with all the sugar to format things for every new prompt was a little annoying, and with the slowdown, by the time a new prompt appeared I'd have almost forgotten the formatting.

Can't blame you for that, editing the Wiki is indeed more exhausting than just posting plain text. 
A tip: I always have a .txt file open in notepad, with the most common functions for formatting, makes copy pasting easier.

I'm also glad to hear from you, yeah the threads slowed down, can't do much about that, but people still are posting and considering both how new and how niche this game is it is somewhat surprising.
If you are fine with it I will grant him access to the Wiki, who knows? Maybe he can format it better so it could be a win. Obviously you can still make any edits you want, no pressures from me.

**Comment by ghost** [comment]
@VBPXKSMI I haven't been working on the wiki for a while, for one because I was sick and also because I was mostly focused on the fork
The biggest issue is definitely that the prompt index isn't clean at all with tons of inconsistencies and not easy to navigate, so I was thinking about, after I'm done with most of the fixes I wanted to work on my fork, doing an overhaul of the page so that it'll be easier for maintainers to add fork, I'll be working on it as late as next week, how does that sound?

**Comment by VBPXKSMI** [comment]
@frowo I will add you as a collaborator here in the meantime, be sure to check your GitHub email and accept the invitation (if you wish), then you can just add whatever prompts you want and when you want to re-work the way everything is formatted you could do so (although showing the changes beforehand would be appreciated).

About the index inconsistencies... are you refering to the new "[Custom Prompts](https://github.com/frowo/LucidTeller/wiki/Custom-Prompts)" page or the old one? Because I don't see how the index could be improved there in the new one, the way the prompts in each section are ordered sure, but not the index per se.
But if you manage a way to maje it clearer do tell.

**Comment by ghost** [comment]
@VBPXKSMI I meant the Custom Prompts page, I meant the inconsistencies especially when it comes to Context and Prompt, they use different formatting most of the time
I'll try to fix things like these, possibly also improve navigation, and add new prompts later, I'll have a pretty busy week soon IRL wise

**Comment by VBPXKSMI** [comment]
@frowo 
> I meant the inconsistencies especially when it comes to Context and Prompt, they use different formatting most of the time

Yeah, this was something we never got quite right, maybe just adding an explanation at the beggining on how it is formatted could help, instead of putting "context and prompt" in every single prompt.

And don't worry about
>I'll have a pretty busy week soon IRL wise

I myself am having the same issue but hopefully at the end of the month will go back to finishing up everything.
